apps/backend-e2e/src/maps.e2e-spec.ts

```diff
@@ -3136,4 +3137,117 @@ describe('Maps', () => {
       afterAll(() => db.cleanup('user', 'mMap', 'run'));
     });
   });
+
+  describe('maps/{mapID}/reviews', () => {
+    describe('GET', () => {
+      let u1, u1token, u2, u2token, mmap, reviewOfficial, reviewUnofficial;
+      beforeAll(async () => {
+        [[u1, u1token], [u2, u2token]] = await Promise.all([
+          db.createAndLoginUser({ data: { roles: Role.MAPPER } }),
+          db.createAndLoginUser({ data: { roles: Role.REVIEWER } })
+        ]);
+
+        mmap = await db.createMap({
+          status: MapStatus.NEEDS_REVISION,
```

Comment:
Use `MapStatusNew` here - current `MapStatus` is just to keep old code compiling, by the end of map submission work it'll be refactored out of the codebase completely. `PUBLIC_TESTING` is most appropriate for this.